### PR TITLE
community/php5: fix dependency (part 2)

### PR DESCRIPTION
**Pull request by vakartel**
some fixes after #5018

removed php5-cli dependency on all extensions
added php5-common (config) dependency to phpdbg

### CODE REVIEW

File: community/php5/APKBUILD [hunk 1]
```diff
@@ -438,6 +439,7 @@ _mv_ext() {
 	local ini=$ext.ini
 	pkgdesc="${ext} extension for PHP"
 	provides="php-$extname"
+	depends=
```
Comment (andypost) [inline community/php5/APKBUILD]: I think better remove it few lines bellow
```diff
-  depends="${pkgname}-common ${depends}"
+  depends="${pkgname}-common"
```

Comment (andypost) [reply community/php5/APKBUILD]: ah.. I'm wrong... it just needs to be
```
depends="${pkgname}-common"
	# extension dependencies
	if [ -n "${2-}" ]; then
 		depends="${2-} ${depends}"
fi
```

Comment (vakartel) [reply community/php5/APKBUILD]: No, we must clean 'depends' variable first. Then populate it with extension-specific depends (if so) and finally add php5-common (config) dependency

Comment (andypost) [reply community/php5/APKBUILD]: Then just make it `${depends}" depends="${2-}` - to append common if any dependencies added

**Pull request closed without merging**